docs/blogs/stabilize_llm_training_cn.md

```diff
@@ -0,0 +1,281 @@
+# DLRover在Kubernetes上千卡级大模型训练稳定性保障的技术实践
```

Comment:
Better to add a hyper link inside  homepage of atorch

Reply:
Done.